### PR TITLE
[DR-1360] Adding nginx configs.

### PR DESCRIPTION
**Pull request by emu47**

### CODE REVIEW

File: nginx-configs/production/image_server_to_iiif.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  }
+  var identitifier = paramsHash['id']
+​
+  if (paramsHash['t'] == 'b') {
+    crop = "square"
+  } else {
+    crop = "full"
+  }
+​
+  var imageUrl = "http://172.16.1.205:8182/iiif/2/"+ identitifier + "/" + crop + "/" + urlSegment +"/0/default.jpg"
```
Comment (bobbywilson0) [inline nginx-configs/production/image_server_to_iiif.js:47]: I thought production was *.195 (and *.194?) and qa *.193?

**Review by bobbywilson0 — APPROVED** [review]
This looks good I did add a question about the IPs because I thought I remember them being different. Also I think you suggested using localhost, which probably makes the most sense.

File: nginx-configs/production/image_server_to_iiif.js (hunk 1 above)
Comment (emu47) [reply nginx-configs/production/image_server_to_iiif.js:47]: You might be right. I was going off information from Brett, but I might have misunderstood. Will confirm with him the right ips tomorrow.

Comment (emu47) [reply nginx-configs/production/image_server_to_iiif.js]: Using local for both now. You are correct. Done.

**Pull request merged**